en/appendices/3-4-migration-guide.rst

```diff
@@ -122,6 +122,18 @@ Cake\Validation\Validator
     * ``provider()``
 Cake\View\StringTemplateTrait
     * ``templates()``
+Cake\View\ViewBuilder
+    * ``templatePath()``
+    * ``layoutPath()``
+    * ``autoLayout()``
```

Comment:
> The following is a list of methods that are deprecated and replaced with ``getX`` and ``setX`` methods:

If people read the above sentence, they would find getAutoLayout and setAutoLayout.